content/800-guides/300-supabase-accelerate.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: 'Set up Supabase with Accelerate Connection Pool'
+metaTitle: "Set up PostgreSQL on Supabase with Prisma Accelerate's Connection Pool"
+description: "Learn how to set up PostgreSQL on Supabase with Prisma Accelerate's Connection Pool"
+sidebar_label: 'Supabase with Accelerate '
+image: '/img/guides/supabase-connection-pooling.png'
+completion_time: '15 min'
+community_section: true
+---
+
+## Introduction
+
+This guides teaches you how to add connection pooling to a PostgreSQL database hosted on [Supabase](https://supabase.com/) using [Prisma Accelerate](/accelerate). 
+
+Prisma Accelerate is a robust and mature connection pooler enabling your database to function properly during traffic spikes and high load scenarios. Check out this [video](https://www.youtube.com/watch?v=cnL75if6Aq0) demonstrating how it performs in a load test or [learn why connection pooling is important](https://www.prisma.io/blog/saving-black-friday-with-connection-pooling).
+
+## Prerequisites
+
+To successfully complete this guide, you need **a connection string for a PostgreSQL instance hosted on Supabase**. It typically looks similar to this:
+
+```bash no-copy
+postgresql://postgres:[YOUR-PASSWORD]@db.nzpppscrldfwlzfalbrf.supabase.co:5432/postgres
+```
+
+If you already have a project using Prisma ORM, you can skip the first two steps and jump ahead to [Step 3. Install the Accelerate extension](#3-install-the-accelerate-extension).
```

Comment:
I think I would personally make this an admonition of some kind. Additionally, should the note here say that if you’re using Prisma ORM _and_ have an Accelerate connection string?